[WIP] Update upstream with medik8s consumers

prabinovRedhat opened the pull request:
Jira link for the epic - [ECOPROJECT-1466](https://issues.redhat.com/browse/ECOPROJECT-1466)
-----

We'd like to increase exposure and trust in our upstream community.

In order to do that we plan to  publish (after having proper permissions) medik8s consumers (End Users) :

in our [upstream website](https://www.medik8s.io/).
[Here](https://kubevirt.io/) is an example of how it was done for KubeVirt
in our [git repo ](https://github.com/medik8s), see an example [Here](https://github.com/kubevirt/kubevirt/blob/main/ADOPTERS.md) of how it was done for KubeVirt

------

prabinovRedhat commented:
@mshitrit

prabinovRedhat committed:
fix

mshitrit commented:
@k-keiichi-rh
@mzeevi 
Can you please review ?

clobrano commented:
Note that we need #61 before being able to test this locally

mshitrit commented:
Hi @batzionb we will also appreciate you feedback on that as a UI expert :smile:

batzionb commented:
I think a more standard approach is to use a [css grid](https://www.w3schools.com/css/css_grid.asp) to accomplish multiple rows of images with the desired layout

batzionb commented:
/lgtm

batzionb commented:
I think it'll look much better just one row with three images, there's no need to take too much space just for three images
Then you also need less css and can make it more simple
Also worth checking if Jekyl has a standard way to embed this in Markdown instead of adding custom html and css

prabinovRedhat committed:
removing NEC and Dana logos temporarily

prabinovRedhat committed:
removing NEC and Dana logos temporarily

prabinovRedhat committed:
add NEC logo

prabinovRedhat commented:
I fixed according to @batzionb comment. As well I added NEC logo after their confirmation.

![Screenshot 2023-09-26 at 16 43 27](https://github.com/medik8s/docs/assets/58551443/3ee2c9e4-6192-4683-a51d-2e1e269bb9e4)

batzionb commented on end_users_section.html:
```diff
@@ -0,0 +1,17 @@
+    <h2>End Users</h2>
+  </div>
+  
+  <div class="logos-container" style="display: flex; flex-direction: row; justify-content: center; align-items: center; margin-bottom: 30px;">
```
Since the styles are defined inline seems to me like you can remove the class attributes

prabinovRedhat replied:
fixed

prabinovRedhat committed:
fix according to comment

prabinovRedhat commented:
sorry for inconvenience , I ran into some problem with my previous PR, I opened new one with all changes.
https://github.com/medik8s/docs/pull/64

cc: @mshitrit
